packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -676,6 +676,25 @@ def _serialize_body_parameter(self, builder: OperationType) -> List[str]:
             retval.append(create_body_call)
         return retval
 
+    def _prepare_params(self, builder: OperationType, is_paging: bool) -> List[str]:
+        if is_paging:
+            return [""]
+        retval = []
+        if builder.parameters.grouped:
```

Comment:
this code will break paging with grouped / flattened parameters because we will no longer ungroup / unflatten them. This will probably also break paging with overloads as well.

Reply:
the default value of `is_paging` is `false` so it will not be skipped in other scenarios. As for paging, I just move them outside `prepare_request` instead of skipping them. Please see here: https://github.com/Azure/autorest.python/blob/85b8ac260a62a2059eb8058ee0ad58d31fae0ae6/packages/autorest.python/autorest/codegen/serializers/builder_serializer.py#L1182